doc/monitoring/getting_started.rst

```diff
@@ -3,66 +3,162 @@
 Monitoring: getting started
 ===========================
 
-.. _monitoring-getting_started-tarantool:
+.. _monitoring-getting_started-install:
 
-Tarantool
----------
+Install metrics rock
+-----------------------
 
-First, install the ``metrics`` package:
+If you are using Tarantool version below 2.11.1 - install the latest version of Metrics. For instructions
+to install the Metrics rock :ref:`view the install documentation <_install-the_usual_way>`.
 
-..  code-block:: console
+.. _monitoring-getting_started-how_to_use:
 
-    $ cd ${PROJECT_ROOT}
-    $ tarantoolctl rocks install metrics
+How to use
+---------------
 
-Next, require it in your code:
+.. hint::
+
+    The use of the module in applications based on the Cartridge framework
+    is considered :ref:`here <_getting_started_cartridge>`.
+
+
+1. Set the node name and enable the output of the standard set of metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
```
Don't use the heading formatting with the lists. If you need to wrap some text as the list item, use the list indentation instead: https://www.tarantool.io/en/doc/latest/contributing/docs/markup/intro/#indentation